x-pack/packages/security-solution/navigation/src/landing_links/landing_links_image_card.tsx

```diff
@@ -0,0 +1,118 @@
+import { getKibanaLinkProps } from './utils';
+
+export interface LandingLinksImageCardProps {
+  item: NavigationLink;
+  urlState?: string;
+  onLinkClick?: (id: string) => void;
+}
+
+const CARD_HEIGHT = 116;
+const CARD_HEIGHT_IMAGE = 106;
+
+const useStyles = () => {
+  const { euiTheme } = useEuiTheme();
+  return {
+    card: css`
+      height: ${CARD_HEIGHT}px;
+      display: flex;
+      justify-content: flex-start;
+    `,
+    cardWrapper: css`
+      height: 100%;
+    `,
+    titleContainer: css`
+      display: flex;
+      align-items: center;
+      height: ${euiTheme.size.l};
+    `,
+    title: css`
+      color: ${euiTheme.colors.primaryText};
+      font-weight: ${euiTheme.font.weight.semiBold};
+    `,
+    description: css`
+      padding-top: ${euiTheme.size.xs};
+      max-width: 550px;
+      font-size: ${euiTheme.size.m};
+    `,
+    imageContainer: css`
+      overflow: hidden;
+      height: 100%;
+      width: ${CARD_HEIGHT_IMAGE}px;
+      flex-grow: 0;
+    `,
+    cardInfoContainer: css`
+      width: min-content;
+    `,
+    image: css`
+      max-inline-size: none;
+      overflow: hidden;
+      height: ${CARD_HEIGHT_IMAGE}px;
+    `,
+  };
+};
+
+const EuiPanelWithLink = withLink(EuiPanel);
+
+export const LandingLinksImageCard: React.FC<LandingLinksImageCardProps> = React.memo(
+  function LandingLinksImageCard({ item, urlState, onLinkClick }) {
+    const styles = useStyles();
+
+    const linkProps = getKibanaLinkProps({ item, urlState, onLinkClick });
+    const { landingImage, title, description, isBeta, betaOptions } = item;
+    return (
+      <EuiFlexItem data-test-subj="LandingImageCard-item" grow={false}>
+        <EuiPanelWithLink {...linkProps} hasBorder paddingSize="xs" css={styles.card}>
+          <EuiFlexGroup
+            gutterSize="s"
+            direction="row"
+            wrap
+            justifyContent="flexStart"
+            alignItems="flexStart"
+            css={styles.cardWrapper}
+          >
+            {landingImage && (
+              <EuiPanel
+                paddingSize="none"
+                hasShadow={false}
+                hasBorder
+                borderRadius="m"
+                css={styles.imageContainer}
+              >
+                <EuiImage
+                  data-test-subj="LandingImageCard-image"
+                  alt={title}
+                  src={landingImage}
+                  css={styles.image}
+                />
+              </EuiPanel>
+            )}
+            <EuiFlexGroup css={styles.cardInfoContainer} gutterSize="s" direction="column">
+              <div css={styles.titleContainer}>
+                <EuiTitle size="xxxs" css={styles.title}>
+                  <h3>{title}</h3>
+                </EuiTitle>
+                {isBeta && <BetaBadge text={betaOptions?.text} />}
+              </div>
+              <span css={styles.description}>{description}</span>
+            </EuiFlexGroup>
```

Comment:
I see `EuiFlexItem`s are missing, and the `div` and `span` with custom CSS should not be necessary. 
The structure should be something like this:

```
<EuiFlexGroup direction="row">
  <EuiFlexItem>
    <EuiPanel><EuiImage ... /><EuiPanel>
  </EuiFlexItem>
  <EuiFlexItem>
    <EuiFlexGroup direction="column">
      <EuiFlexItem>
        <EuiTitle ...>{title}</EuiTitle>
      </EuiFlexItem>
      <EuiFlexItem>
        <EuiText ...>{description}</EuiText>
      </EuiFlexItem>
    </EuiFlexGroup>
  </EuiFlexItem>
</EuiFlexGroup>
```